content/en/docs/concepts/overview/what-is-kubernetes.md

```diff
@@ -48,7 +48,7 @@ Containers have become popular because they provide extra benefits, such as:
 * Resource isolation: predictable application performance.
 * Resource utilization: high efficiency and density.
 
-## Why you need Kubernetes and what can it do
+## Why you need Kubernetes and what it can do
```

Comment:
This will preserve inbound links:
```suggestion
## Why you need Kubernetes and what it can do {#why-you-need-kubernetes-and-what-can-it-do}
```

Alternatively, this:
```suggestion
## Why you need Kubernetes and what it can do {#why-you-need-kubernetes}
```
breaks inbound links but makes future breaking less likely.

Reply:
Ah sorry, I forgot about links, but yes, that works.

Reply:
Cool. @ChrisChinchilla are you willing to revise the PR?